frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyMfaCodeHandler.java

```diff
@@ -410,10 +407,6 @@ private APIGatewayProxyResponseEvent handleSuccess(
                 codeRequest.getMfaMethodType().getValue(),
                 journeyType);
 
-        sessionService.storeOrUpdateSession(
-                session.setCurrentCredentialStrength(CredentialTrustLevel.MEDIUM_LEVEL),
-                userContext.getAuthSession().getSessionId());
-
```

Comment:
Have we followed through here that nothing else about the session is updated in this call?